datafusion/core/tests/parquet/filter_pushdown.rs

```diff
@@ -65,7 +65,12 @@ fn generate_file(tempdir: &TempDir, props: WriterProperties) -> TestParquetFile
     test_parquet_file
 }
 
-#[tokio::test]
+// The result can be flaky if `rand` crate updated.
+//
+// Reason: Data generated by
```

Comment:
These fuzzy tests based random data gen become non relevant because of `rand` version upgrade and seems the seed now returns different random, different from what tests expect in terms of data distribution, column data, etc making the test useless. Considering the Filter Pushdown is important test and conditions in the tests are carefully chosen to cover use cases, we probably materialize the parquet file and get rid of random data gen to avoid test to be stale when `rand` crate upgraded.

@mbrobbel WDYT?

Reply:
I noticed the same (in https://github.com/apache/datafusion/pull/14447) for some different tests, which I fixed by explicitly using `rand_chacha`: https://github.com/apache/datafusion/pull/14447/files#diff-4a599584dfc900ec21169f4f820a1b1db46b004b77533dab83a6178d5d3a467eR4253 following https://rust-random.github.io/book/crate-reprod.html.